aix360/algorithms/rbm/boolean_rule_cg.py

```diff
@@ -126,7 +126,7 @@ def fit(self, X, y):
         v, zNew, Anew = beam_search(r, X, self.lambda0, self.lambda1,
                                     K=self.K, UB=UB, D=self.D, B=self.B, eps=self.eps)
 
-        while (v < -self.eps).any() and (self.it < self.iterMax) and (time.time()-self.starttime < timeMax):
+        while (v < -self.eps).any() and (self.it < self.iterMax) and (time.time()-self.starttime < self.timeMax):
```

Comment:
@Marleen1 I also added `self.` here in front of `timeMax` as I believe it's needed. If it looks fine to you, I will merge the pull request.

Reply:
Yes, that looks good!